src/lib/featureFlagSlice.ts

```diff
@@ -0,0 +1,41 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+// const API_ADDRESS = "http://localhost:8080";
+
+interface FeatureFlagsState {
+  enableAbout: boolean;
+  enableWiki: boolean;
+  enableFAQ: boolean;
+
+  // Add more feature flags as needed
+}
+
+export const initialState: FeatureFlagsState = {
+  enableAbout: false,
+  enableWiki: false,
+  enableFAQ: false,
+
+  // Initialize other feature flags
+};
```

Comment:
A design choice to be had: should these be enabled by default? And then only disabled if the API says so...

Reply:
Sure, that makes sense